content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+<br /> <br />
+
+## Configure the Registry Server in the Palette Console
+After pushing the pack, configure the registry server in the Palette console so that you can access the pack in the Palette. Here are the steps to follow.  
+
+- First, login to the Palette console, and switch to the Tenant admin view.
+
+
+- As a Tenant admin, navigate to **Admin Settings &#8594; Registries &#8594; Pack Registries**.
+
+
+- Click the **Add New Pack Registry** and enter the pack registry name, endpoint, and user credentials. For a consistent experience in this tutorial, we have chosen the name **private-pack-registry**, as shown in the visual below. Though you can choose any other name per your organization's naming convention. Lastly, click **Confirm** to finsh adding a registry.
+
+  <br />
+
+  ![Visual of registry server edit option in Palette Tenant admin settings.](/tutorials/create-pack/registries-and-packs_create-pack_registry-edit.png)
+
+  <br /> <br />
+
+
+- The Palettle syncs the registry server periodically. However, you can sync it manually by clicking the **Three Dots** against the registry server name and selecting the **Sync** option.  
+
+  <br />
+
+  ![Visual of registry server sync in the Palette](/tutorials/create-pack/registries-and-packs_create-pack_registry-sync.png)
+
+  <br /> <br />
+
+
+
+# Create a Profile and Deploy a Cluster
+After configuring the registry server in Palette, create a Profile and deploy it to a cluster using either of the following two methods:
+
+1. Using the Palette console.
+
+2. Using Terraform code.
+
+As part of this tutorial, you will create a Profile for the **AWS cloud**. However, you can choose any other cloud service provider, provided the following two items are configured:
+<br /> 
+
+1. **Cloud account**: A cloud account added to your Palette project settings. This tutorial assumes the AWS cloud account name is **spectro-cloud**. The below visual shows how to add and verify a cloud account to your project by navigating to **Project Settings  &#8594; Cloud Accounts  &#8594; AWS  &#8594; Add AWS Account** in the Palette console. 
+
+
+<br />
+
+![Visual of Cloud Accounts in the Palette console.](/tutorials/create-pack/registries-and-packs_create-pack_palette-cloud-account.png)
+
+<br /> <br />
+
+
+2. **SSH key**: An SSH key created in the region where you will deploy the cluster. This tutorial will deploy the cluster in the **us-east-2** region and assumes an SSH key, **aws_key_sk_us_east_2**, exists already in the AWS console. 
+
+<br /> <br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Palette Console" key="Palette_Console">
+
+
+## Create a Profile
+Switch to the **Default** view and select the **Profile** section in the left **Main Menu** to create a full profile that combines core infrastructure and add-on layers. Click on the **Add Cluster Profile** button, and provide the details in the wizard that follows. The wizard will present you with several sections. 
+<br />
+
+### Section: Basic Information
```

Comment:
This pattern of a colon in a title is no something we do. You can remove Section: and just leave the title.